src/main/kotlin/com/jetbrains/snakecharm/lang/psi/impl/refs/SmkSectionReference.kt

```diff
@@ -2,34 +2,65 @@ package com.jetbrains.snakecharm.lang.psi.impl.refs
 
 import com.intellij.codeInsight.lookup.LookupElementBuilder
 import com.intellij.openapi.util.TextRange
+import com.intellij.psi.PsiElement
 import com.intellij.psi.PsiReferenceBase
 import com.intellij.psi.util.PsiTreeUtil
 import com.intellij.util.PlatformIcons
 import com.jetbrains.python.psi.PyStringLiteralExpression
+import com.jetbrains.snakecharm.codeInsight.SnakemakeSectionsInShellReferenceContributor
 import com.jetbrains.snakecharm.lang.psi.SmkRuleOrCheckpoint
 
 
 class SmkSectionReference(
         element: PyStringLiteralExpression,
-        textRange: TextRange,
-        private val sectionName: String
+        private val textRange: TextRange,
+        private val sectionName: String?
 ) : PsiReferenceBase<PyStringLiteralExpression>(element, textRange) {
     private val key: String = element.text.substring(textRange.startOffset, textRange.endOffset)
 
-    override fun resolve() =
+    override fun resolve(): PsiElement? {
+        return if (sectionName != null &&
+                element.text.substring(textRange.startOffset, textRange.endOffset) != sectionName) {
             getKeywordArguments().firstOrNull { it.name == key }
```

Comment:
Doesn't work if input includes results of `unpack` method.

use https://github.com/JetBrains-Research/chipseq-smk-pipeline as an example for testing.

![image](https://user-images.githubusercontent.com/72933/62640259-dc393a80-b949-11e9-9783-0b7ad1d05aa3.png)